build-system/app.js

```diff
@@ -731,7 +731,13 @@ app.get(['/examples/*.html', '/test/manual/*.html'], (req, res, next) => {
   });
 });
 
+function escapeRegExp(string) {
+  return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Comment:
Based on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

Reply:
This change actually broke the regex.

What warning did we see?

Reply:
@lannka See the last link in the PR description.